src/main/java/com/google/ehub/data/UserDataManager.java

```diff
@@ -0,0 +1,28 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import java.io.IOException;
+
+public class UserDataManager {
+  public UserProfile getUserProfile(String email) {
+    ProfileDatastore data = new ProfileDatastore();
```

Comment:
I think the name ```data``` might be a little too generic, something like ```profileDatastore``` might be better.